From 098858f74e62849a205f7d35ccc4ebb7ec3daf35 Mon Sep 17 00:00:00 2001
From: Philipp Trulson <philipp@trulson.de>
Date: Thu, 10 Jan 2019 19:32:46 +0100
Subject: [PATCH 1/1] Disable installer warning

---
 bin/vagrant | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/bin/vagrant b/bin/vagrant
index 0e6abdcef..cb36e9cf9 100755
--- a/bin/vagrant
+++ b/bin/vagrant
@@ -155,11 +155,6 @@ begin
     end
   end

-  if !Vagrant.in_installer? && !Vagrant.very_quiet?
-    # If we're not in the installer, warn.
-    env.ui.warn(I18n.t("vagrant.general.not_in_installer") + "\n", prefix: false)
-  end
-
   # Acceptable experimental flag values include:
   #
   # Unset  - Disables experimental features
--
2.20.1
